picard/webservice/api_helpers.py

```diff
@@ -78,35 +78,36 @@ def host(self):
     def port(self):
         return self._port
 
-    def get(self, path_list, handler, priority=False, important=False, mblogin=False,
-            cacheloadcontrol=None, refresh=False, queryargs=None, parse_response_type=DEFAULT_RESPONSE_PARSER_TYPE):
+    def url_from_path_list(self, path_list):
         path = self.api_path + "/".join(path_list)
-        return self._webservice.get(self.host, self.port, path, handler,
-                                    priority=priority, important=important, mblogin=mblogin,
-                                    refresh=refresh, queryargs=queryargs, parse_response_type=parse_response_type)
-
-    def post(self, path_list, data, handler, priority=False, important=False,
-             mblogin=True, queryargs=None, parse_response_type=DEFAULT_RESPONSE_PARSER_TYPE,
-             request_mimetype=None):
-        path = self.api_path + "/".join(path_list)
-        return self._webservice.post(self.host, self.port, path, data, handler,
-                                     priority=priority, important=important, mblogin=mblogin,
-                                     queryargs=queryargs, parse_response_type=parse_response_type,
-                                     request_mimetype=request_mimetype)
-
-    def put(self, path_list, data, handler, priority=True, important=False,
-            mblogin=True, queryargs=None, request_mimetype=None):
-        path = self.api_path + "/".join(path_list)
-        return self._webservice.put(self.host, self.port, path, data, handler,
-                                    priority=priority, important=important, mblogin=mblogin,
-                                    queryargs=queryargs, request_mimetype=request_mimetype)
-
-    def delete(self, path_list, handler, priority=True, important=False,
-               mblogin=True, queryargs=None):
-        path = self.api_path + "/".join(path_list)
-        return self._webservice.delete(self.host, self.port, path, handler,
-                                       priority=priority, important=important, mblogin=mblogin,
-                                       queryargs=queryargs)
+        return host_port_to_url(self.host, self.port, path=path)
+
+    def get(self, path_list, handler, **kwargs):
```

Comment:
This move breaks tests, but I'm not sure why. I guess that's because the prototype changed, as tests using MagickMock

Reply:
See my comment above: The tests break because the APIHelper got a completely changed interface.